force-app/main/default/classes/DonationHistoryController_TEST.cls

```diff
@@ -165,7 +163,7 @@ public class DonationHistoryController_TEST {
         System.assertEquals(1, donationHistory.size(), 'donationHistory should contain only one element');
     }
 
-    private static Contact findContactByLastname(String firstname) {
+    private static Contact findContactByFirstName(String firstName) {
         for (Contact c : [SELECT ID, Firstname
                           FROM Contact]) {
             if (c.Firstname == firstname) {
```

Comment:
I'd really like to see these test methods use mocked Contact data rather than SOQL in the future here, that way we won't have to rely on finding a contact with the correct first name.

```suggestion
            if (c.FirstName == firstName) {
```